### ISSUE TITLE
video latency

### ISSUE THREAD

**Issue by mtncgj123** [dji-sdk/Tello-Python#87]
Hello, this project seems good, but when I run the code, there is serious video latency, the image will blur for 2 seconds and restore and blur again with the error below
![image](https://user-images.githubusercontent.com/92588069/183800033-a0fe2f62-fa93-4d1f-97ec-c746be0b3479.png)
 can anyone helps?

**Comment by juanmill4** [comment]
hii 
you resolve?

**Comment by mtncgj123** [comment]
@juanma-t3j  Yeah, I tried to use the latest robotmaster SDK (https://github.com/dji-sdk/RoboMaster-SDK) and received more stable video stream

**Comment by 1411734304** [comment]
@mtncgj123 Can I have a look at your code? I want to learn it, because my current code ability is very weak。
Thank you very much and look forward to your reply。

**Comment by mtncgj123** [comment]
> @mtncgj123 Can I have a look at your code? I want to learn it, because my current code ability is very weak。 Thank you very much and look forward to your reply。

Sure, you can leave me with your email address or contact me with my email in my profile

**Comment by 1411734304** [comment]
@mtncgj123 My email address is 1411734304@qq.com. Thank you very much for your help. If it doesn't work, you can try jingqihu38@gmail.com
